Bio/SeqAnalysis/SeqAnalyzer.py

```diff
@@ -0,0 +1,196 @@
+import numpy as np
+from Bio.SeqUtils.ProtParam import ProteinAnalysis, IsoelectricPoint
+
+
+class SeqAnalyzer:
+    """
+    A class that takes in records from SeqAnalysis.database, performs
+    multiple operations on it and returns 2 dictionaries with result values. For further information about
+    said dictionaries please see 'results()' function documentation.
+    The operations that the class performs upon received data from SeqAnalysis.database is it:
+    1. Calculates molecular weight of protein sequences
+    2. Calculates isoelectric point for protein sequences
+    3. Counts standard aa in protein sequences, in a list of dicts (amino_acids)
+    4. Calculates aromaticity for protein sequences according to Lobry 1994 method
+    5. Calculates amino acid percentage content based aa content in protein sequences
+    6. Calculates instability indexes for protein sequences according to Guruprasad et al 1990
+    7. Calculates flexibility of protein sequences according to Vihinen, 1994
+    8. Calculates fraction of helix, turn and sheet for protein sequence
+    9. Calculates gravy of protein sequences according to Kyte and Doolittle
+    10. Calculates molar extinction coefficients for protein sequences
+    11. Calculates median length of sequences
+    12. Calculates average length of sequences
+    13. Calculates standard deviation od sequence length
+    14. Calculates 'N' and 'C' terminus amino acid occurrences frequencies
+    """
+
+    def __init__(self, data: dict):
```

Comment:
Parameter typing is ok, but if you want to indicate what types are object in `dict` you may want to use `Dict[str, SeqRecord]` next time.